app/views/help/spark/helpGcCpuTimeHeuristic.scala.html

```diff
@@ -0,0 +1,12 @@
+<p>The ratio of jvmGcTime to executorRunTime is checked, to see if GC is taking too much time (providing more memory could help) or too little time (memory may be over provisioned, and can be reduced).</p>
```

Comment:
Rephrase this. Help page should be simple and more intuitive for the users.

```
GC Time Heuristic flags jobs which are spending too much time on Garbage Collection.

If GC is taking too much time, then providing more memory could help. On the other hand, if GC is taking too little time, it could mean that the memory is over provisioned and it can be reduced.
```
 
Remove all the threshold values from the help page. I don't think it will help the users in ay way.